### ISSUE TITLE
Programmatically close the notyf?

### ISSUE THREAD

**Issue by olanchuy** [caroso1222/notyf#20]
Hi, great work on this. Is there a way to close programmatically the notyf?

ex. var notyf = new Notyf();
// NoTe: should return some id 
var theID = notyf.confirm();

after some time...
notyf.close(theID);

**Comment by caroso1222** [comment]
Hey!

Thanks for bringing this to my attention. There's currently no way to programmatically close the Notyf. I'll work on this feature and add it on v3.x.

**Comment by caroso1222** [comment]
v3.0.0 has been released with a lot of cool things. I left this for 3.1.0 as I didn't really got time to add this feature, but it's on the roadmap for sure.

**Comment by azhararmar** [comment]
Close option would be good, is it coming anytime soon? a button to dismiss would also be good.

**Comment by bharatramnani94** [comment]
Hey @caroso1222 
Currently, the `success`, `error` and `open` methods do not return anything.
Maybe they can return the `notification` (of type `NotyfNotification`), which can be passed to a new `dismiss(notificationToDismiss: NotyfNotification)` method to achieve this functionality.
Let me know what you think. Maybe I can take this issue.

**Comment by caroso1222** [comment]
Hey @bharatramnani94. Yeah, I like that. Thanks for bringing it up again.

I'm still not sure whether to go functional on this one like a method `dismiss(notification)` on the global instance or OOP-style in the notification instance `myNotyf.dismiss()`.

I'm inclined towards the former as I don't think a notification should be able to call itself dismissed, only the master should. (Just thinking out loud)

I think, then, a good solution would be something like:

```typescript
const notyf = new Notyf();

const foobar = notyf.success('baz');

setTimeout(() => notyf.dismiss(foobar), 5000);
```

I'm totally fine if you want to take it, thanks!

Right off the bat, I think renaming  `_removeNotification` to `dismiss` and making it public would make the trick, but will need testing. It might be a breaking change in case someone is using that private, undocumented method so maybe for backward compatibility purpose we'd just keep it and write a new _proxy_ method `dismiss`.

**Comment by caroso1222** [comment]
Closed in https://github.com/caroso1222/notyf/pull/63. Credits to @bharatramnani94.

**Comment by bharatramnani94** [comment]
> Closed in #63. Credits to @bharatramnani94.

Honestly, all credit goes to you @caroso1222. You did the heavy work for it. I'm just glad that this got done.
